core/broadleaf-profile/src/main/resources/bl-profile-applicationContext-entity.xml

```diff
@@ -23,9 +23,9 @@
            
     <!-- Entity mappings - override for extensibility -->
     <bean id="org.broadleafcommerce.profile.core.domain.Address" class="org.broadleafcommerce.profile.core.domain.AddressImpl" scope="prototype">
-        <property name="country">
-            <bean class="org.broadleafcommerce.profile.core.domain.CountryImpl">
-                <property name="abbreviation" value="US"/>
+        <property name="isoCountryAlpha2">
```

Comment:
I think this can be reverter ? as we will not delete CountryImpl

Reply:
It cannot see the property with name "country" I think the field Country was removed with State in AddressImpl. I think in this case it would be better to leave the changed code